microcosm_sqlite/dumpers/csv.py

```diff
@@ -14,21 +14,27 @@ class CSVDumper:
     def __init__(
         self,
         graph,
-        model_cls,
-        bulk_mode=False,
+        store,
+        model_cls=None,
     ):
         self.graph = graph
-        self.model_cls = model_cls
+        self.store = store
+        self.model_cls = model_cls or store.model_class
         self.defaults = dict()
 
     def default(self, **kwargs):
         self.defaults.update(kwargs)
         return self
 
-    def dump(self, fileobj, items=None):
+    def dump(self, fileobj, items=None, fieldnames=None, extrasaction=None):
```

Comment:
Why not `field_names` or `extras_action`?

Reply:
We could, but these are the names that DictWriter is expecting so thought it made sense to keep it consistent

Reply:
I guess we can define our own API. Ok, will change it